development-docs/TESTING.md

```diff
@@ -181,6 +181,14 @@ All available test groups are listed in [Constants](systemtest/src/main/java/io/
 ## Environment variables
 
 System tests can be configured by several environment variables, which are loaded before test execution.
+Variables can be defined via env or via configuration file, this file can be placed anywhere and path to this
+file is defined by env variable `CONFIG`, if `CONFIG` env variable is not defined, default config location is used `systemtest/config.json`.
+
+Loading of variables has following priority:
+1. Environment variable
+2. Variable defined in config
+3. Default value
```

Comment:
```suggestion
Variables can be defined via environmental variables or a configuration file, this file can be located anywhere on the file system as long as a path is provided to this file. 
The path is defined by environmental variable `SYSTEM_TEST_CONFIG_PATH`, if the `SYSTEM_TEST_CONFIG_PATH` environmental variable is not defined, the default config file location is used `systemtest/config.json`.

Loading of system configuration has the following priority order:
1. Environment variable
2. Variable defined in configuration file
3. Default value
```
I recommend CONFIG -> `SYSTEM_TEST_CONFIG_PATH` as it's less likely to conflict with pre-set envars on running systems.